interface/src/avatar/MyAvatar.h

```diff
@@ -413,8 +415,19 @@ class MyAvatar : public Avatar {
     Q_PROPERTY(float walkBackwardSpeed READ getWalkBackwardSpeed WRITE setWalkBackwardSpeed NOTIFY walkBackwardSpeedChanged);
     Q_PROPERTY(float sprintSpeed READ getSprintSpeed WRITE setSprintSpeed NOTIFY sprintSpeedChanged);
     Q_PROPERTY(bool isInSittingState READ getIsInSittingState WRITE setIsInSittingState);
+
+    /**jsdoc
+    * @deprecated This property is deprecated and will be removed. If you need it, please contact the developers.
+    */	
     Q_PROPERTY(MyAvatar::SitStandModelType userRecenterModel READ getUserRecenterModel WRITE setUserRecenterModel);
```

Comment:
```suggestion
    Q_PROPERTY(MyAvatar::SitStandModelType userRecenterModel READ getUserRecenterModel WRITE setUserRecenterModel);  // Deprecated
```
This JSDoc won't generate any documentation. The deprecation is documented adequately above. A simple comment after the property declaration suffices.
Similarly, elsewhere.